checkbox-core-snap/series16/snap/snapcraft.yaml

```diff
@@ -57,6 +57,14 @@ package-repositories:
     ppa: hardware-certification/public
 
 parts:
+  version-calculator:
+    plugin: dump
+    source: .
+    override-pull: |
+      # version.txt created by prepare.sh
```

Comment:
```suggestion
      # The version string is written to a file by prepare.sh as a pre-build step
      # since it is computed based on git repo metadata not available for a snapcraft build
```